src/app/phast/phast-report/results-data/results-data.component.html

```diff
@@ -175,16 +175,25 @@ <h4 class="show-print print-section-header">Results Data</h4>
         </td>
       </tr>
 
-      <tr>
-        <td>Annual CO2 Emissions (tonne CO<sub>2</sub>)</td>
+      <tr *ngIf="!inReport" class="tr-underline">
+        <td>Annual CO<sub>2</sub> Emissions (kg CO<sub>2</sub>/hr)</td>
         <td [ngClass]="{'selected-modification': selectedModificationIndex == -1}">{{baseLineResults.co2EmissionsOutput | sigFigs:'3'}}</td>
         <td *ngFor="let mod of modificationResults; let index = index;" [ngClass]="{'selected-modification': index == selectedModificationIndex}">
           <span *ngIf="mod.co2EmissionsOutput">{{mod.co2EmissionsOutput | sigFigs:'3'}}</span>
           <span *ngIf="!mod.co2EmissionsOutput">&mdash; &mdash;</span>
         </td>
       </tr>
-      <tr class="tr-underline">
-        <td>Annual CO2 Emissions Savings (tonne CO<sub>2</sub>)</td>
+
+      <tr *ngIf="inReport">
+        <td>Annual CO<sub>2</sub> Emissions (kg CO<sub>2</sub>/hr)</td>
```

Comment:
Some places say kg/hr, some still say tonne. I'm assuming that's the intent, with houry vs annual units differing